databases/week1/data.sql

```diff
@@ -0,0 +1,58 @@
+--Find out how many tasks are in the task table
+
+SELECT * FROM TASK;
+SELECT COUNT(id) FROM TASK;
+
+--Find out how many tasks in the task table do not have a valid due date
+
+SELECT count(title) FROM TASK WHERE (due_date IS Null);
+
+--Find all the tasks that are marked as done => What does that mean?
+
+SELECT TASK.title, STATUS.NAME FROM TASK
+LEFT JOIN STATUS ON TASK.STATUS_ID = STATUS.ID
+WHERE STATUS.NAME = 'done';
```

Comment:
since we want all tasks we used left join since task is the left table however,  all other JOINs would've worked in this case. 
besides that, I left a comment asking what does that mean since there was nothing with "Done" in the task table and I spent some time to understand where to get it. this exercise was blurry to me.
another solution I'd do the following:  
SELECT * FROM task
WHERE status_id = 3;
(I had to look into other tables to find the word (Done).

Reply:
Good job for finding the table with status text. It is just a way that database is designed. You can use normal join and there is no need to specify left join.